### ISSUE TITLE
Playground doesn't show results

### ISSUE THREAD

**Issue by richb-hanover** [PRQL/prql#3054]
### What happened?

Using the 0.9.0 release, the official [Playground](https://prql-lang.org/playground/) does not show any results in the right-hand pane (nothing for the generated SQL or the SQL results)

Even abbreviating the PRQL query to `from invoices` there is no output.

### PRQL input

```elm
from invoices
```

### SQL output

```SQL
-- none --
```

### MVCE confirmation

- [X] Minimal example
- [ ] New issue

**Comment by richb-hanover** [comment]
Hmmm... I bet this is related to #3047. Closing it

**Comment by eitsupi** [comment]
Let it keep opened.

**Comment by max-sixty** [comment]
Thanks @richb-hanover . We're urgently trying to fix now given we just hit HN again.

I don't think it's #3047 — playground doesn't rely on crates.io.

**Comment by max-sixty** [comment]
Looks like the same error as https://github.com/PRQL/prql/pull/2521 — we could try reverting any updates since then?

**Comment by max-sixty** [comment]
https://github.com/PRQL/prql/pull/3071 is one thing, but I think unlikely to be it

**Comment by max-sixty** [comment]
Not sure this is fixed

**Comment by max-sixty** [comment]
Super weird — it seems to work for some folks, like @snth https://discord.com/channels/936728116712316989/945832228011798528/1133498403414880399

I think maybe it was only broken on Firefox?

**Comment by snth** [comment]
Yes, it's been working fine for me on Chrome.

**Comment by richb-hanover** [comment]
Just now (2230 on 25 Jul 2023 UTC) I saw errors with both Chrome 116.0.5845.42  and Firefox 115.0.2 on macOS 12.6.7. Both Chrome and Firefox showed substantially the same errors.

The Playground displayed SQL output for the default (complicated) PRQL query. I did not check it to see if it made sense. The SQL output for a simple `from invoices` was reasonable. The comment showed  "PRQL version 0.9.2"

However, the "output.arrow" for the default PRQL query displayed `Error: Binder Error: Referenced column "customer_id" not found in FROM clause! Candidate bindings: "invoices.column0"` , and the `from invoices` query showed this for the SQL result:

```
column0
--
[object Response]
```
Finally, I see these errors in the Firefox console:
```
Could not create web worker(s). Falling back to loading web worker code in main thread, which might cause UI freezes. Please see https://github.com/microsoft/monaco-editor#faq [simpleWorker.js:20:16](https://prql-lang.org/playground/playground/static/node_modules/monaco-editor/esm/vs/base/common/worker/simpleWorker.js)

You must define a function MonacoEnvironment.getWorkerUrl or MonacoEnvironment.getWorker [simpleWorker.js:22:12](https://prql-lang.org/playground/playground/static/node_modules/monaco-editor/esm/vs/base/common/worker/simpleWorker.js)

Object { timestamp: Date Tue Jul 25 2023 18:29:13 GMT-0400 (Eastern Daylight Time), level: 2, origin: 4, topic: 4, event: 4, value: "WITH table_1 AS (\n  SELECT\n    customer_id,\n    total - 0.8 AS _expr_0,\n    invoice_date\n  FROM\n    invoices\n),\ntable_0 AS (\n  SELECT\n    COALESCE(SUM(_expr_0), 0) AS sum_income,\n    COUNT(*) AS ct,\n    customer_id\n  FROM\n    table_1\n  WHERE\n    _expr_0 > 5\n    AND invoice_date >= DATE '2010-01-16'\n  GROUP BY\n    customer_id\n)\nSELECT\n  c.customer_id,\n  CONCAT(c.last_name, ', ', c.first_name) AS name,\n  table_0.sum_income,\n  table_0.ct,\n  version() AS db_version\nFROM\n  table_0\n  JOIN customers AS c ON table_0.customer_id = c.customer_id\nORDER BY\n  table_0.sum_income DESC\nLIMIT\n  10\n\n-- Generated by PRQL compiler version:0.9.2 (https://prql-lang.org/)\n" }
[log.ts:70:20](https://prql-lang.org/playground/playground/static/node_modules/@duckdb/duckdb-wasm/src/log.ts)

```
I hope this is helpful.

**Comment by eitsupi** [comment]
Thanks @richb-hanover, I can repro that.
![image](https://github.com/PRQL/prql/assets/50911393/8a90180c-2527-41c9-9618-270ccd24ecd4)
![image](https://github.com/PRQL/prql/assets/50911393/6c13cfb2-24f7-4c28-905b-ae07deee0beb)

**Comment by max-sixty** [comment]
Thanks a lot @richb-hanover .

~I think https://github.com/PRQL/prql/pull/3079 fixes — I can repro locally~

**Comment by max-sixty** [comment]
This is actually still broken — I must have missed something when whizzing through earlier. 

Is it anything to do with how we upload the data? It looks like we have it in a different representation...

**Comment by aljazerzen** [comment]
Finally closed by #3102

**Comment by richb-hanover** [comment]
I agree - it looks fixed to me. Thanks!
